cdap-ui/app/cdap/components/KeyValuePairs/KeyValuePairs.less

```diff
@@ -0,0 +1,65 @@
+/*
+* Copyright © 2016 Cask Data, Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License"); you may not
+* use this file except in compliance with the License. You may obtain a copy of
+* the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+* WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+* License for the specific language governing permissions and limitations under
+* the License.
+*/
+
+.namespace-preferences {
+  width: 100%;
+  padding-right: 20px;
+  height: 100%;
+  max-height: 250px;
+  overflow-y: auto;
+  overflow-x: hidden;
+}
+
+.name-value-container {
+  .fa-trash {
+    margin-right: 20px;
+  }
+}
+
+.key-value-pair-preference {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-right: -25px;
+
+  input {
+    border-bottom: 1px solid lightgrey;
+  }
+
+  input:focus {
+    outline: 0;
+  }
+}
+
+.key-input,
+.value-input {
+  border: none;
+  width: 40%;
+  margin-right: 10px;
+}
+
+.add-row-btn,
+.remove-row-btn {
+  padding-right: 10px;
+  cursor: pointer;
+}
+
+.add-row-btn {
+  color: black;
+}
+.remove-row-btn {
+  color: red;
+}
```

Comment:
Nest all these inside a parent container, these changes are applied globally for the same class name